docs/UserGuide.md

```diff
@@ -461,15 +474,15 @@ Format: `unarchiveT INDEX`
 
 ### Listing tasks with deadlines up to and including the specified date: `remindT`
 
-Lists tasks in YellowBook with deadlines up to and including the specified date.
+Lists tasks in task list with deadlines up to and including the specified date.
 
 Format: `remindT DEADLINE`
 
 * Both completed and incomplete tasks are listed.
 
 * Task with deadlines that are already past are also listed.
 
-<div markdown="span" class="alert alert-info">:bulb:Tasks that are complete are also listed so users are reminded to delete them.
+<div markdown="span" class="alert alert-info">:bulb:Tasks that are complete are also listed so users are reminded to delete or archive them.
 </div>
 
```

Comment:
Add incorrect usage box with bulletpoint
"Do not adhere to deadline constraints. (link to section 2 intro)

Reply:
wait sorry dont link to section 2 intro, instead just write it must be dd-mm-yyyy format since its just one field